tracer/src/Datadog.Trace/Telemetry/Transports/TelemetryTransportStrategy.cs

```diff
@@ -45,7 +45,7 @@ public static IApiRequestFactory GetAgentIntakeFactory(ImmutableExporterSettings
                 Log.Information("Using {FactoryType} for telemetry transport, with UDS path {Path}.", nameof(SocketHandlerRequestFactory), settings.TracesUnixDomainSocketPath);
                 return new SocketHandlerRequestFactory(new UnixDomainSocketStreamFactory(settings.TracesUnixDomainSocketPath), TelemetryHttpHeaderNames.GetDefaultAgentHeaders(), GetBaseEndpoint());
 #elif NETCOREAPP3_1_OR_GREATER
-                Log.Information<string, string, int>("Using {FactoryType} for telemetry transport, with Unix Domain Sockets path {Path} and timeout {Timeout}ms.", nameof(UnixDomainSocketStreamFactory), settings.TracesUnixDomainSocketPath, settings.TracesPipeTimeoutMs);
+                Log.Information<string, string>("Using {FactoryType} for telemetry transport, with Unix Domain Sockets path {Path} and timeout {Timeout}ms.", nameof(UnixDomainSocketStreamFactory), settings.TracesUnixDomainSocketPath);
```

Comment:
This string template expected 3 values but there are only 2. The value for `{Timeout}` is missing.

Reply:
```suggestion
                Log.Information<string, string>("Using {FactoryType} for telemetry transport, with Unix Domain Sockets path {Path}.", nameof(UnixDomainSocketStreamFactory), settings.TracesUnixDomainSocketPath);
```